### PR TITLE
fixed ironic spelling error

### PR DESCRIPTION
**Pull request by Autre31415**
@RocketChat/core 

title says it all

### CODE REVIEW

File: app/scripts/preload.js [hunk 1]
```diff
@@ -287,7 +287,7 @@ try {
 		}
 	});
 } catch(e) {
-	console.log('Spellchecker unavailble');
+	console.log('Spellchecker unavailable');
```
Comment (kethinov) [inline app/scripts/preload.js]: Spell checker is two words bro.

File: app/scripts/preload.js [hunk 2]
```diff
@@ -287,7 +287,7 @@ try {
 		}
 	});
 } catch(e) {
-	console.log('Spellchecker unavailable');
+	console.log('spellchecker module unavailable');
```
Comment (kethinov) [inline app/scripts/preload.js]: 👍

Comment (kethinov) [inline app/scripts/preload.js]: this also has the added benefit of making this PR seven billions times moar better than this PR https://github.com/RocketChat/Rocket.Chat.Electron/pull/202

Comment (xenithorb) [inline app/scripts/preload.js]: Seven BILLION? Damn that's harsh bro

**Comment by rodrigok** [comment]
Haha, thanks

**Pull request merged**